packages/devtools-plugin/src/index.ts

```diff
@@ -23,15 +23,12 @@ const postMessage = (message: DevMessage) => {
 const devToolsPlugin = <State>(): ProdoPlugin<
```

Comment:
I'm not sure this plugin needs to be parameritezed by the `State` anymore. I know that the State type is used for the `DevToolsUniverse`, but the plugin accesses more than just the state now.

Reply:
The problem is that if I remove all this, state stops being typed for projects that use the devtools. Happy to show you if it helps.